build-system/pr-check.js

```diff
@@ -212,28 +212,34 @@ const command = {
     let docFiles = files.filter(isDocFile);
     timedExecOrDie(`${gulp} check-links --files ${docFiles.join(',')}`);
   },
-  buildRuntime: function() {
+  runPreBuildChecks: function() {
     timedExecOrDie(`${gulp} clean`);
     timedExecOrDie(`${gulp} lint`);
+  },
+  buildRuntime: function() {
+    timedExecOrDie(`${gulp} clean`);
     timedExecOrDie(`${gulp} build`);
```

Comment:
This function (buildRuntime) is called before the unit tests and the integration tests. I've already added a TODO to investigate why the unit tests currently need this function to be called. So this is done, I believe.